src/main/java/org/fenixedu/academic/ui/struts/action/commons/student/enrollment/bolonha/AbstractBolonhaStudentEnrollmentDA.java

```diff
@@ -127,6 +127,9 @@ public ActionForward enrolInDegreeModules(ActionMapping mapping, ActionForm form
             addActionMessage("error", request, ex.getKey(), ex.getArgs());
 
             return prepareShowDegreeModulesToEnrol(mapping, form, request, response, bolonhaStudentEnrollmentBean);
+        } catch (org.fenixedu.bennu.core.domain.exceptions.DomainException de) {
```

Comment:
The other place in this class where a DomainException is used could be replaced by this type